helm/v2/templates/deployment.yaml

```diff
@@ -68,12 +68,18 @@ spec:
           containerPort: {{ .Values.service.ports.metrics }}
           protocol: TCP
         livenessProbe:
+          periodSeconds: {{ .Values.probes.liveness.periodSeconds }}
```

Comment:
can't we just do a `{{toYaml}}` here so they can pass the fields according to the k8s api?